v2/clients/http/utils/common.go

```diff
@@ -150,6 +150,10 @@ func sendRequest(ctx context.Context, req *http.Request) ([]byte, errors.EdgeX)
 	}
 
 	// Handle error response
+	if resp.StatusCode == http.StatusNotFound {
```

Comment:
the condition should be `resp.StatusCode >= http.StatusBadRequest`, because they are all error status

Reply:
Modified.
Return error when resp.StatusCode > http.StatusMultiStatus:
```
	if resp.StatusCode <= http.StatusMultiStatus {
		return bodyBytes, nil
	}

	// Handle error response
	msg := fmt.Sprintf("request failed, status code: %d, err: %s", resp.StatusCode, string(bodyBytes))
	errKind := errors.KindMapping(resp.StatusCode)
	return nil, errors.NewCommonEdgeX(errKind, msg, nil)
```